.github/workflows/autodocs-images.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build Images Reference Docs
+on:
+  workflow_dispatch:
+env:
+  WORKDIR: images
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out Destination Repo"
+        uses: actions/checkout@v2
+        with:
+          path: edu
+
+      - name: "Check out Images Monorepo"
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
```

Reply:
I will update the version after merging these.